stylesheets/_touch-friendly-links.scss

```diff
@@ -0,0 +1,10 @@
+// This mixin will expand the clickable area for a link on mobile and tablet to make it easier to click on a touch
+// device.
+
+@mixin touch-friendly-links($padding: 3px) {
+  a {
+    padding: $padding;
+    margin: -1 * $padding;
+    outline: none;
```

Comment:
I've just tried @nickcolley 's suggestion. Looks fine in my browser, but again I don't really know what the use case is here, or how to test it.